src/blocks/OurTeam/index.js

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
+import styled from 'styled-components'
+import PreTitle from 'components/PreTitle'
+import Grid from 'components/Grid'
+import get from 'utils/get'
+import Typography from 'components/Typography'
+import shuffle from 'utils/shuffle'
+import TeamMemberList from './TeamMemberList'
+
+const Section = styled.section`
+  background-color: ${get('colors.valhalla.700')};
+  margin: 0 -16px;
+  padding: 72px 16px;
+
+  @media (min-width: ${get('breakpoints.md')}) {
+    margin: 0 -32px;
+    padding: 88px 32px;
+  }
+  @media (min-width: ${get('breakpoints.lg')}) {
+    padding: 128px 0;
+  }
+  @media (min-width: ${get('breakpoints.xl')}) {
+    padding: 168px 0;
+  }
+`
+
+const Content = styled.div`
+  grid-column: 1 / -1;
+  display: flex;
+  flex-direction: column;
+`
+
+const Title = styled(Typography)`
+  color: ${get('colors.white')};
+  margin-top: 4px;
+`
+
+const Carousel1 = styled(TeamMemberList)`
+  margin-top: 48px;
+  @media (min-width: ${get('breakpoints.md')}) {
+    margin-top: 56px;
+  }
+  @media (min-width: ${get('breakpoints.lg')}) {
+    margin-top: 72px;
+  }
+  @media (min-width: ${get('breakpoints.xl')}) {
+    margin-top: 96px;
+  }
+`
+
+const Carousel2 = styled(TeamMemberList)`
+  margin-top: 42px;
+  @media (min-width: ${get('breakpoints.md')}) {
+    margin-top: 56px;
+  }
+  @media (min-width: ${get('breakpoints.lg')}) {
+    margin-top: 80px;
+  }
+`
+
+const OurTeam = ({ ourTeam, ...props }) => {
+  const [carousel1, setCarousel1] = React.useState([])
+  const [carousel2, setCarousel2] = React.useState([])
```

Comment:
can we call them `aboveCarousel` and `beneathCarousel` to understand more their purpose?

Reply:
I prefer keeping the name as is, if it's okay for you. We don't know if some day we won't have a 3rd carousel coming beneath, and with `above` / `beneath` it leaves less flexibility :)